site/src/docs/components/highlight/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/components/highlight'
+title: 'Highlight'
+navTitle: 'Highlight'
+---
+
+import { Highlight } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview style={{ backgroundColor: 'var(--color-black-5)' }}>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." />
+</PlaygroundPreview>
+
+<PlaygroundPreview style={{ backgroundColor: 'var(--color-black-5)' }}>
```

Comment:
Maybe add smaller headers before the components to make it which is which. `#### Highlight` ... `#### Quote`

Reply:
I think this might still be good? To differentiate clearly which is which.